packages/backend-plugin-api/src/services/system/types.ts

```diff
@@ -38,6 +39,8 @@ export type ServiceRef<
    */
   scope: TScope;
 
+  multiton: TInstances extends 'multiton' ? true : false;
```

Comment:
Hm how breaking is this really, when the field is required? I mean when people have mixed versions of plugin-api in their project, will the consuming side be confused by the missing multiton field returned by old versions of createServiceRef, leading to type errors?

Reply:
👍 switched to optional and added a test